### PR TITLE
[kor] Adds phonetic whitelist

### PR DESCRIPTION
**Pull request by yeonju123**
- [ ] Updated `Unreleased` in `CHANGELOG.md` to reflect the changes in code or data.

Korean phonetic whitelist is created,`kor_phonetic.whitelist`

### CODE REVIEW

File: data/wikipron/whitelist/kor_phonetic.whitelist [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+u
+ɰ
+a̠ː
+k̚ #allophone of /k/.
+ɦ #allophone of /h/.
+p̚ #allophone of /p/.
+t͈ 
+ʝ #allophone of /h/.
+s͈
+t̚ #allophone of many alveolar, alveoli-palatal consonants on coda position.
```
[the remark this answers is not in the document]
Comment (yeonju123) [reply data/wikipron/whitelist/kor_phonetic.whitelist:38]: > If you choose, you can rerun the scrape for (just) Korean as per the instructions in `data/wikipron/src` and commit the filtered lists. PR #154 should make that somewhat easier to do going forward. If you don't, I'll just do that as a follow-up.

I will work on this :)

**Comment by kylebgorman** [comment]
Looks good, one nit.

Can you do the steps here to give us a sense of how many words are filtered by this? If you commit the summary TSV files it'll be apparent. It'll also print out all the words that get filtered, so if you missed anything important...

https://github.com/kylebgorman/wikipron/blob/readme/data/wikipron/whitelist/README.md

**Comment by yeonju123** [comment]
After I ran `generate_summary.py`, it updated `README.md` as well. Do I commit this change as well? I do not see that in the guidelines, but I just wanted to confirm :)

**Comment by kylebgorman** [comment]
Yes, please do. That update will let us see how many words actually got
deleted!

So you know, that particular `README.md` renders as a nice, human-readable
table with the number of entries per language.

**Comment by kylebgorman** [comment]
Looks good, so you filter about 250 words with this. (I am seeing this by comparing the README before and after your change).

Whitelists look good modulo those two comments on the "comment" style, then this is ready to go.

**Comment by yeonju123** [comment]
whitelist style is already fixed, so we can merge it, I think.

File: data/wikipron/whitelist/kor_phonetic.whitelist [hunk 2]
```diff
@@ -0,0 +1,60 @@
+e̞
+n
+d͡ʑ #allophone of /j/.
```
[the remark this answers is not in the document]
Comment (yeonju123) [reply data/wikipron/whitelist/kor_phonetic.whitelist]: >Whitelists look good modulo those two comments on the "comment" style, then this is ready to go.

The style of `whitelist` is already fixed, so I think we can merge it.

**Comment by yeonju123** [comment]
> Looks good, so you filter about 250 words with this. (I am seeing this by comparing the README before and after your change).
> 
> Whitelists look good modulo those two comments on the "comment" style, then this is ready to go.

The style is already fiex, so we can merge, I think :)

**Pull request merged**